src/rpc/server.cpp

```diff
@@ -425,6 +426,21 @@ static inline JSONRPCRequest transformNamedArguments(const JSONRPCRequest& in, c
         } else {
             hole += 1;
         }
+        if (out.params.size() == 0) skipped += 1;
+    }
+    // If leftover "args" param was found, use it as a source of positional
+    // arguments and add named arguments after.
+    auto positional_args = argsIn.find("args");
```

Comment:
nit 1: call it `_args` to have some internal "feeling"?

nit 2: forbid registering arguments with this name.

Reply:
Outside of this changeset, it may not be clear where "args" comes from. It may be good to add a comment here that "args" is set in rpc/client.cpp::RPCConvertNamedValues().

Reply:
> I think it's fine if RPC methods want to handle positional arguments with their own logic, and don't think it would be good to add extra code to detect and forbid it a priori.

It makes this code needlessly complex. If in the future someone wants to add an "args" arg, they can modify the code.